bower.json

```diff
@@ -28,7 +28,7 @@
     "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
     "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0"
+    "polymer": "Polymer/polymer#^1.1.0"
```

Comment:
No, I changed to [new style recommendations of Polymer 1.1](https://blog.polymer-project.org/announcements/2015/08/13/1.1-release/).